agave_app/CameraWidget.cpp

```diff
@@ -1,157 +1,38 @@
 #include "CameraWidget.h"
 #include "RenderSettings.h"
 
-QCameraWidget::QCameraWidget(QWidget* pParent, QCamera* cam, RenderSettings* rs)
+#include "qtControls/controlFactory.h"
+
+#include "renderlib/uiInfo.hpp"
+#include "renderlib/CameraUiDescription.hpp"
+
+#include <QLabel>
+
+QCameraWidget::QCameraWidget(QWidget* pParent, RenderSettings* rs, CameraDataObject* cdo)

```

Comment:
now all this class has to do is produce the ui info for each property of the camera, and make calls into the controlFactory to create the ui... as seen below.  A sort of ideal world would be where this is just a generic loop over all the properties and the uiinfos.